crates/aptos/src/common/types.rs

```diff
@@ -1163,25 +1164,33 @@ pub struct MovePackageDir {
 
     /// ...or --compiler COMPILER_VERSION
     /// Specify the version of the compiler.
-    /// Defaults to `1`, unless `--move-2` is selected.
+    /// Defaults to the latest stable compiler version
```

Comment:
Perhaps note that this is `(at least 2)`, since if you're not a compiler developer you don't know that 2 is stable.